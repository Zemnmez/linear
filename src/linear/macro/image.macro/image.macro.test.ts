import pluginTester from 'babel-plugin-tester';
import plugin from 'babel-plugin-macros';

pluginTester({
    plugin,
    snapshot: true,
    babelOptions: { filename: __filename},
    tests: [
        `import macro from '../image.macro';
        const img = macro('test.png', {
            width: 1, height: 10
        });`
    ]
})