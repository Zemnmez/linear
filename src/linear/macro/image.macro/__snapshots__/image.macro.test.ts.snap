// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`macros 1. macros: 1. macros 1`] = `

import macro from '../image.macro';
        const img = macro('test.png', {
            width: 1, height: 10
        });

      ↓ ↓ ↓ ↓ ↓ ↓

const img = {
  blob:
    "/9j/2wBDAAYEBQYFBAYGBQYHBwYIChAKCgkJChQODwwQFxQYGBcUFhYaHSUfGhsjHBYWICwgIyYnKSopGR8tMC0oMCUoKSj/2wBDAQcHBwoIChMKChMoGhYaKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCgoKCj/wAARCAAKAAEDASIAAhEBAxEB/8QAFgABAQEAAAAAAAAAAAAAAAAAAAcI/8QAHhAAAQIHAQAAAAAAAAAAAAAAAAIEBQcVGFRVlNH/xAAUAQEAAAAAAAAAAAAAAAAAAAAA/8QAFBEBAAAAAAAAAAAAAAAAAAAAAP/aAAwDAQACEQMRAD8AoFuUtsN53LBm2uxfav8AoX6AP//Z",
  width: 1,
  height: 10,
};


`;
